src/template.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  ARROW_CLASS,
+  BACKDROP_CLASS,
+  BOX_CLASS,
+  CONTENT_CLASS,
+  SVG_ARROW_CLASS,
+} from './constants';
+import {div, isElement} from './dom-utils';
+import {Instance, PopperChildren, PopperElement, Props} from './types';
+import {arrayFrom} from './utils';
+
+const innerHTML = (): 'innerHTML' => 'innerHTML';
```

Comment:
Could use an explanation comment. I remember it had something to do with terser, but not sure what exactly

Reply:
🤷‍♂ it's for Firefox extensions. They don't allow `.innerHTML` but aren't smart enough to detect this lol

Reply:
Then let's write a comment about it.